workers/unity/Packages/com.improbable.gdk.playerlifecycle/Systems/PlayerCreation/SendCreatePlayerRequestSystem.cs

```diff
@@ -31,43 +35,54 @@ protected override void OnCreateManager()
                 ComponentType.ReadOnly<WorkerEntityTag>(),
                 ComponentType.ReadOnly<OnConnected>()
             );
+        }
 
-            playerSpawnGroup = GetComponentGroup(
-                ComponentType.ReadOnly<ShouldRequestPlayerTag>()
-            );
+        public void QueuePlayerCreationRequest(Vector3 spawnPosition = default, byte[] serializedArguments = null)
+        {
+            playerCreationRequested = true;
+            this.spawnPosition = Vector3f.FromUnityVector(spawnPosition);
+            this.serializedArguments = serializedArguments;
         }
 
         protected override void OnUpdate()
         {
-            var initEntities = initializationGroup.GetEntityArray();
-            for (var i = 0; i < initEntities.Length; ++i)
+            if (PlayerLifecycleConfig.AutoRequestPlayerCreation)
```

Comment:
If I want to auto create a player at startup - but with arguments, is this useful for me? And if not, is this misleading as an option?

Reply:
you can now define what default position/serialized arguments to use with auto-player creation